js/main.js

```diff
@@ -244,6 +325,9 @@ function fillWeaponWheel(scroller, limit, isOffhand) {
 
 function fillAshesWheel(scroller, limit) {
     const usableAshes = collectUsableAshNames();
+    if(usableAshes.length == 0){
```

Comment:
Match the surrounding code style:
```suggestion
    if (usableAshes.length === 0) {
```